### DISCUSSION TITLE
zero confirmation transaction

### DISCUSSION THREAD

**Discussion by arvin-lau** [btcsuite/btcd#2247] [Q&A]
What is the btcd zero confirmation transaction limit? Is it 25?

**Comment by guggero** [comment]
Doesn't look like that limit is currently implemented: https://github.com/btcsuite/btcd/blob/ff2e03e11233fa25c01cf4acbf76501fc008b31f/mempool/mempool.go#L1340

**Reply by arvin-lau** [reply]
Thank you for you reply
